### PR TITLE
Update with latest Parchment

### LINKED ISSUE #124: Update interpreters
- [x] Basic stylehint support
  - [ ] Finalise stylehint protocol
- [x] Garglk text formatting support
- [ ] Update ZVM to use the new functions
- [x] Update Quixe to use the new functions?
- [x] Update html generation
- [ ] Transcript recording

### PR DESCRIPTION
**Pull request by curiousdannii**
Big PR: new terps!

Closes #124, #148

### CODE REVIEW

File: IFComp/lib/IFComp/Schema/Result/Entry.pm [hunk 1]
```diff
@@ -1044,133 +1027,40 @@ sub _create_parchment_page {
...
-</div>
-
+    <div id="gameport">
+        <div id="windowport">
+            <noscript>
+                <p>You'll need to turn on Javascript in your web browser to play this game.</p>
+            </noscript>
+        </div>
+        <div id="loadingpane">
+            <img src="/static/interpreter/waiting.gif" alt="LOADING"><br>
```
Comment (curiousdannii) [inline IFComp/lib/IFComp/Schema/Result/Entry.pm:1054]: Just had an idea: it shouldn't be hard to make it show the entry's cover image here, but I don't know what the substitution would be. Should I make a new issue?

**Comment by jmacdotorg** [comment]
Thanks for this PR!

The purpose of the changes to the Entry.pm library is not clear to me; I would expect that updates to the interpreters should be simple "drop-in" replacements. Could you please explain that a bit?

**Comment by curiousdannii** [comment]
They're not drop in because the interpreters are all new! The Parchment interpreter uses a completely new display library. Quixe is now instantiated through Parchment. And it's usigg ES modules.

File: IFComp/lib/IFComp/Schema/Result/Entry.pm [hunk 2]
```diff
@@ -1226,24 +1117,12 @@ EOF
...
     my $self = shift;
 
-    if ( $self->is_zcode ) {
-        return <<EOF;
-<script src="/static/interpreter/parchment/jquery.min.js"></script>
-<script src="/static/interpreter/parchment/parchment.min.js"></script>
-<script src="/static/interpreter/transcript_recorder/if-recorder.js"></script>
-<link rel="stylesheet" type="text/css" href="/static/interpreter/parchment/parchment.css">
-<link rel="stylesheet" type="text/css" href="/static/interpreter/parchment/style.css">
+    return <<EOF;
```
Comment (curiousdannii) [inline IFComp/lib/IFComp/Schema/Result/Entry.pm:1120]: I couldn't tell if I could just delete this whole function, but it's no longer being called from _create_parchment_page

**Comment by jmacdotorg** [comment]
These changes make the entry-processing tests fail. (Which stands to reason!) [See this test report.](https://travis-ci.org/github/iftechfoundation/ifcomp/builds/717091688?utm_source=github_status&utm_medium=notification) Could you please update these tests as appropriate? (And let me know if you need some additional guidance here.)

**Comment by curiousdannii** [comment]
Fixed those tests, and TidyAll!

**Comment by jmacdotorg** [comment]
This is excellent work, @curiousdannii. (And don't mind the coveralls thing, that's an automated indicator which I'm happy to overrule when warranted.)

I'm going to merge this. Thank you for all your hard work here.

**Pull request merged**